src/handler/src/commands/github/issue-pr.ts

```diff
@@ -426,8 +440,10 @@ export class IssuePRLookup implements Command {
 		return decision.toLowerCase().replace(/_/g, ' ');
 	}
 
-	private static formatCommentString(num: number): string {
-		return `${num} comment${num > 1 ? 's' : ''}`;
+	private static formatCommentString(number: number, locale: string): string {
```

Comment:
There is already functionality for plurals in i18next:
https://www.i18next.com/translation-function/plurals

Maybe this can be useful.